frontend/app/components/Dashboard/components/FilterSeries/FilterSeries.tsx

```
@@ -1,182 +1,236 @@
 import ExcludeFilters from './ExcludeFilters';
-import AddStepButton from "Components/Dashboard/components/FilterSeries/AddStepButton";
-import {Button, Space} from "antd";
-import {ChevronDown, ChevronUp, Trash} from "lucide-react";
+import { Button, Space } from 'antd';
+import { ChevronDown, ChevronUp, Trash } from 'lucide-react';
 
-
-const FilterCountLabels = observer((props: { filters: any, toggleExpand: any }) => {
+const FilterCountLabels = observer(
+  (props: { filters: any; toggleExpand: any }) => {
     const events = props.filters.filter((i: any) => i && i.isEvent).length;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)